cardano.json

```diff
@@ -1881,17 +1881,16 @@
 
   , "Utxo":
```

Comment:
As a side-note, this definition should probably not even be here. This is not used anywhere in the Cardano schema actually. It is used in a few places in Ogmios but that's not so-to-speak needed to capture Cardano's on entities (as weird as it may sound).